lib/post/widgets/post_view.dart

```diff
@@ -84,32 +117,180 @@ class PostSubview extends StatelessWidget {
           MediaView(
             scrapeMissingPreviews: scrapeMissingPreviews,
             post: post,
-            postView: postViewMedia,
+            postView: widget.postViewMedia,
             hideNsfwPreviews: hideNsfwPreviews,
             markPostReadOnMediaView: markPostReadOnMediaView,
             isUserLoggedIn: isUserLoggedIn,
           ),
-          if (postViewMedia.postView.post.body != null)
+          if (widget.postViewMedia.postView.post.body != null)
             Padding(
-              padding: const EdgeInsets.symmetric(vertical: 8.0),
+              padding: const EdgeInsets.only(top: 8.0),
               child: CommonMarkdownBody(
                 body: post.body ?? '',
               ),
             ),
+          if (showCrossPosts && sortedCrossPosts.isNotEmpty)
+            Padding(
```

Comment:
It might be good to separate this into its own widget just for organization purposes (the widget can reside in this file, but just moved out of the PostView widget - to see what I mean by this, you can check out `Tagline` on the FeedPage)

Reply:
All set to go now!